contracts/libraries/UniswapV2OracleLibrary.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity >=0.5.0;
+
+import '@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol';
+import '@uniswap/lib/contracts/libraries/FixedPoint.sol';
+
+// library with helper methods for oracles that are concerned with computing average prices
+library UniswapV2OracleLibrary {
+    using FixedPoint for *;
+
+    // helper function that returns the current block timestamp within the range of uint32, i.e. [0, 2**32 - 1]
+    function currentBlockTimestamp() internal view returns (uint32) {
+        return uint32(block.timestamp % 2 ** 32);
+    }
+
+    // helper function that returns the time that has elapsed since the given block timestamp in uint32
+    function timeElapsedSince(uint32 blockTimestampLast) internal view returns (uint32) {
```

Comment:
this isn't used anywhere, we should delete it until we need it, or use it in `currentCumulativePrices`

Reply:
since it's fairly trivial my vote would be to not use it, though it's nice insofar as it makes the overflow-safe nature of the 32-bit arithmetic clear